src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/AbstractEntityAIGuard.java

```diff
@@ -438,6 +440,11 @@ private IAIState checkAndAttackTarget()
         if (fighttimer > 0)
         {
             fighttimer--;
+            if (fighttimer == 0)
```

Comment:
I sthis enough to have this here, what happens if combat finishes and they are hungry, or try to retreat with low health?

Reply:
ye because this is the point which also prevents the interruption of the other AI's

Reply:
kk